website/content/configuration/calico.md

```diff
@@ -3,17 +3,30 @@ title: Issues with Calico
 weight: 20
 ---
 
-Simple setups with [Calico](https://docs.projectcalico.org/) don't
-require anything special, you can just install and configure MetalLB
-as usual and enjoy.
+## The easy way
 
-However, if you are using
-Calico's
-[external BGP peering capability](https://docs.projectcalico.org/v3.0/usage/configuration/bgp) to
-advertise your cluster prefixes over BGP, and also want to use BGP in
-MetalLB, you will need to jump through some hoops.
+As of Calico 3.18 (from early 2021), Calico now supports limited
+integration with MetalLB.  Calico can be configured to announce
+LoadBalancer IPs itself via BGP.  Simply run MetalLB *without* BGP
+turned on, and then configure the IP range(s) that you want Calico
+to announce.
 
-## The problem
+Example:
+
+```
+$ calicoctl patch BGPConfig default --patch '{"spec": {"serviceLoadBalancerIPs": [{"cidr": "10.11.0.0/16"},{"cidr":"10.1.5.0/24"}]}}'
+```
+
+Be aware that Calico announces the entire CIDR block provided, not
+individual LoadBalancer IPs.  If you need to announce more specific
+routes, then explicitly list them in `serviceLoadBalancerIPs`.
```

Comment:
Based on https://projectcalico.docs.tigera.io/networking/advertise-service-ips#advertising-service-ips-quick-glance correct way to avoid this would be using `externalTrafficPolicy=Local` on services. However I have not tested it (yet).

Reply:
Ah, yeah, `externalTrafficPolicy=Local` looks right.  I can't verify right now, but I'll try to check tomorrow.

Reply:
I just tried `externalTrafficPolicy=Local` for one service and immediately saw ~1/6th failures and added latency, so I turned it back off.  BGP announcements looked much better, so this is probably just a bug in my config, but I'm not going to be able to debug it tonight.